apps/web/app/page.tsx

```diff
@@ -51,9 +51,12 @@ export default function Page(): JSX.Element {
           onClose={() => setActiveRun(undefined)}
         />
       )}
-      <section className=" overflow-scroll relative h-full">
+      <section
+        className="overflow-scroll relative"
+        style={{ height: "calc(100vh - 40px)" }}
```

Comment:
does this work when run details section is opened ?

Reply:
Updated, the top nav was transparent